paper/user_study.tex

```diff
@@ -46,8 +46,123 @@ \subsection{Methodology}\label{subsec:methodology}
 We analyse the results which we get from the survey and improve our application afterward.
 
 \subsection{Results}
-% todo: show specific cases of user study. Use table of feedbacks and resolutions
-% todo: not sure how we name this section.
+To evaluate the user study, we get results from interviews.
+We conducted five user studies in total and summarize the correctness and incorrectness of answers into the table, as shown in \ref{table:Results}.
```

Comment:
How did you evaluate correct or not? Because you should describe it here, it is not clear for me. Did you mark it as wrong when the first answer given was not the correct one or how? Or did you mark it wrong when the participant was only able to solve it when we gave a hint?

Reply:
I mark it wrong when the first answer they give was wrong.
Yes should I right a standard how we evalute correctness and in correctness? 
It's a bitte ambiguous